src/apprt/gtk/builder_check.zig

```diff
@@ -0,0 +1,43 @@
+const std = @import("std");
+const build_options = @import("build_options");
+
+pub const c = @cImport({
+    @cInclude("gtk/gtk.h");
+    if (build_options.adwaita) {
+        @cInclude("libadwaita-1/adwaita.h");
+    }
+});
+
+pub fn main() !void {
+    var gpa = std.heap.GeneralPurposeAllocator(.{}){};
+    const alloc = gpa.allocator();
+
+    const filename = filename: {
+        var it = try std.process.argsWithAllocator(alloc);
+        defer it.deinit();
+
+        _ = it.next() orelse return error.NoFilename;
+        break :filename try alloc.dupeZ(u8, it.next() orelse return error.NoFilename);
+    };
+    defer alloc.free(filename);
+
+    const data = try std.fs.cwd().readFileAlloc(alloc, filename, std.math.maxInt(u16));
+    defer alloc.free(data);
+
+    if ((comptime !build_options.adwaita) and std.mem.indexOf(u8, data, "lib=\"Adw\"") != null) {
+        std.debug.print("{s}: skipping builder check because Adwaita is not enabled!\n", .{filename});
```

Comment:
Maybe it would be useful to print the filename/path here?

Reply:
I am, aren't I?